.github/workflows/system-libs.yaml

```diff
@@ -48,6 +48,7 @@ jobs:
            for dir in /usr/lib /usr/lib64; do                       \
              for file in $(find $dir -type f -name "*.so.*"); do    \
                if ! expr "$file" : ".*\.hmac" >/dev/null 2>&1; then \
+                 if [[ $f =~ "libclang-cpp.so" ]]; then continue; fi\
```

Comment:
github actions use /bin/sh by default which disables bash constructs like `[[ ... ]]`.  You'll need to use expr like on line 50.